CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# [Milestone 2] Matching service

- Completed matching service 

To test matching service: 
- Run backend:  cd `matching-service` and run `node index`
- Run frontend: cd `frontend` and run `npm start` 
- Open 2 `localhost:3000/selectquestiondifficulty` tabs 

-  Select the same question difficulty on both tabs 
- Check that the room that both users are directed to has the same roomID 
- Check that the user will be directed back to the match selection page after clicking on the `leave-room` button
